examples/timeseries_peak_finding.py

```diff
@@ -0,0 +1,78 @@
+"""
+==============================
+Find Peaks in SunPy TimeSeries
+==============================
+
+This example illustrates how to find minimum or maximum peaks in a TimeSeries.
+"""
```

Comment:
I also would like you to add some warning or note that peak finding is a complex problem that has many potential solutions and this example is just one method of many.

Reply:
Sure, I will add that.